### PR TITLE
Added New Components

### PR DESCRIPTION
**Pull request by wtseng21**
Added the following
- Icon
- Popover
- Tooltip
- UncontrolledFade

### CODE REVIEW

File: src/index.d.ts [hunk 1]
```diff
@@ -1,78 +1,82 @@
...
+import TabPane from './TabPane.svelte';
+import Toast from './Toast.svelte';
+import ToastBody from './ToastBody.svelte';
+import ToastHeader from './ToastHeader.svelte';
+import Tooltip from './Tooltip.svelte';
+import UncontrolledAlert from './UncontrolledAlert.svelte';
+import UncontrolledButtonDropdown from './UncontrolledButtonDropdown.svelte';
+import UncontrolledCollapse from './UncontrolledCollapse.svelte';
+import UncontrolledFade from './UncontrolledFade.svelte';
+import UncontrolledDropdown from './UncontrolledDropdown.svelte';
```
Comment (bestguy) [inline src/index.d.ts:79]: Is this switch to adding `.svelte` correct?

My understanding is this is importing the `*.d.ts` definitions for each compnent and seems to behave correctly in build & editor.

**Review by bestguy — COMMENTED** [review]
Thank you for adding!  Sorry we missed.  However I have a question on adding .svelte in the index.d.ts

**Comment by bestguy** [comment]
Hi @wtseng21 , sorry I had soe time this morning and jumped ahead and added d.ts here #229 
I'm happy to consider adding the .svelte suffix in question above if I'm missing something.

**Comment by wtseng21** [comment]
Hi @bestguy,
The PR was originally to add missing components such as `<Icon />` to the declaration file.
The `.svelte` suffix was oversight on my end and wasn't intended to be part of the switch.
I saw your changes so I'll go ahead and close this PR. Thanks!

**Comment by bestguy** [comment]
Thank you for the fixes!

**Pull request closed without merging**